vendor/github.com/docker/docker/vendor.conf

```diff
@@ -118,9 +118,9 @@ github.com/googleapis/gax-go v2.0.0
 google.golang.org/genproto 694d95ba50e67b2e363f3483057db5d4910c18f9
 
 # containerd
-github.com/containerd/containerd 9b32062dc1f5a7c2564315c269b5059754f12b9d # v1.2.1
+github.com/containerd/containerd aa5e000c963756778ab3ebd1a12c67449c503a34 # v1.2.1+
 github.com/containerd/fifo 3d5202aec260678c48179c56f40e6f38a095738c
-github.com/containerd/continuity bd77b46c8352f74eb12c85bdc01f4b90f69d66b4
+github.com/containerd/continuity 004b46473808b3e7a4a3049c20e4376c91eb966d
```

Comment:
Ah, sorry, I looked at this PR, and wanted to check if we needed to update these as well (as they're out of date), but I can bring it in sync with containerd in https://github.com/docker/cli/pull/1561